app/src/main/java/org/simple/clinic/home/overdue/phonemask/PhoneMaskBottomSheetController.kt

```diff
@@ -3,11 +3,13 @@ package org.simple.clinic.home.overdue.phonemask
 import io.reactivex.Observable
```

Comment:
Either remove the following function in this class and inline it in `patientPhoneNumberStream()` as its not being reused anymore,

```
 private fun sheetCreated(events: Observable<UiEvent>) =
events.ofType<PhoneMaskBottomSheetCreated>()
```

or reuse it in `setupView()`.